locations/spiders/nobis_printen_de.py

```
@@ -0,0 +1,52 @@
+        # blocker, so the iframe HTML is only available base64-encoded in a
+        # data attribute rather than as a live iframe src.
+        if map_blob := response.css(
+            "[data-borlabs-cookie-content-blocker-id=google-maps]::attr(data-borlabs-cookie-content)"
+        ).get():
+            decoded = base64.b64decode(map_blob).decode()
+            if src := re.search(r'src="([^"]+)"', decoded):
+                item["lat"], item["lon"] = url_to_coords(src.group(1))
+
+        days = response.css(".nobis-hours__day::text").getall()
+        times = response.css(".nobis-hours__time::text").getall()
+        hours_string = ", ".join(f"{day.strip()} {time.strip()}" for day, time in zip(days, times))
```

Ruff (0.16.2)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)